jaeger_client/config.py

```diff
@@ -246,6 +248,14 @@ def tags(self):
                 tags[key.strip()] = value.strip()
         return tags
 
+    @property
+    def extra_codecs(self):
```

Comment:
This feels rather misleading, it's not adding "extra" codecs, it's overriding (partially) the existing ones.

I would much rather use the approach we took in https://github.com/jaegertracing/jaeger-client-java/pull/296 where we have just a single config property `codecs`, which is a comma-separated list like `jaeger,b3` or `jaeger,w3c`. We don't actually need to support the list right now, just single value `b3` that will replace the jaeger codecs.

Reply:
It seems the API is a little different for Tracer in the Java client and here, should I rename this parameter to codecs and add a new optional argument to the Tracer `__init__` method, or even rename the existing `extra_codecs` param that is already there? I was kinda just piggybacking on that being present to add the codec in a simple way

Reply:
I think we don't need to change the Tracer API, and you can still pass the B3 codec as `extra_codecs` to it. I was mostly referring to the API of the Configuration class, it's there I would like to have `codecs=x,y,z` property.